### ISSUE TITLE
Cannot use batman(theme) in oh-my-fish after installed

### ISSUE THREAD

**Issue by MITSUBOSHI** [oh-my-fish/theme-batman#2]
## Problems

Cannot use batman(theme) in oh-my-fish after installed 
## Execution Environment

-fish 2.2.0
-Oh My Fish! 1.0.0
-OS X El Capitan (10.11.4)
## Steps to reproduce the problem

1: `% omf install batman`
    Console says -> theme batman successfully installed.

2:  

``` log
fish: Unknown command 'fst'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): fst
                                                            ^
in command substitution
    called on line 0 of file ~/.local/share/omf/themes/batman/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input

fish: Unknown command 'off'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): off
                                                            ^
in command substitution
    called on line 0 of file ~/.local/share/omf/themes/batman/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input

fish: Unknown command 'snd'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): snd
                                                            ^
in command substitution
    called on line 0 of file ~/.local/share/omf/themes/batman/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input

fish: Unknown command 'off'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): off
                                                            ^
in command substitution
    called on line 0 of file ~/.local/share/omf/themes/batman/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input

fish: Unknown command 'fst'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): fst
                                                            ^
in command substitution
    called on line 0 of file ~/.local/share/omf/themes/batman/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input

fish: Unknown command 'off'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): off
                                                            ^
in command substitution
    called on line 0 of file ~/.local/share/omf/themes/batman/fish_prompt.fish

in function 'fish_prompt'
    called on standard input

in command substitution
    called on standard input

sed: option requires an argument -- e
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
fish: Unknown command 'off'
~/.local/share/omf/themes/batman/fish_prompt.fish (line 1): off
```

The errors like above occurred.
Even if I removed the theme, the errors show every time I type “omf” command.
## Expect

Can work batman correctly and use it in oh-my-fish
Never show the errors
## Related Issue

https://github.com/oh-my-fish/theme-batman/issues/1

p.s.: I will report you again when I find the reason of this issue. Thank you in advance!!

**Comment by elvishp2006** [comment]
+1 same issue here.

**Comment by fredoliveira** [comment]
If you end up here by chance, the fix is to `omf reload` after the theme installation.

**Comment by palindrom615** [comment]
If my PR #5 were to be accepted we could close this issue :]

**Comment by mfournial** [comment]
This error can be cause too when you already have a definition of the greeting function.
I fixed it going into my original `config.fish` and removing my original `fish_greeting` that was still here.

**Comment by bobthecow** [comment]
This should be fixed by #5
